cmd/argocd/commands/admin/backup_test.go

```diff
@@ -362,6 +372,430 @@ status: {}
 	}
 }
 
+func Test_importResources(t *testing.T) {
+	type args struct {
+		bak  string
+		live string
+	}
+
+	tests := []struct {
+		name                     string
+		args                     args
+		applicationNamespaces    []string
+		applicationsetNamespaces []string
+		prune                    bool
+		skipResourcesWithLabel   string
+		stopOperation            bool
+		overrideOnConflict       bool
+	}{
+		{
+			name: "It should update live object if skip label is not present in backup object",
```

Comment:
```suggestion
			name: "It should update live object if skip label is not present in backup object",
```
Please add a better test name for this. Rn, it doesn't clear what's the objective of the test case.

Reply:
Sure, will update the name of the test 👍